cli/f3.go

```diff
@@ -316,8 +316,8 @@ var f3SubCmdCerts = &cli.Command{
 				f3FlagOutput,
 			},
 			Before: func(cctx *cli.Context) error {
-				if count := cctx.NArg(); count > 1 {
-					return fmt.Errorf("too many arguments: expected at most 1 but got %d", count)
+				if cctx.NArg() != 1 {
+					return IncorrectNumArgs(cctx)
 				}
```

Comment:
This one allows zero too, look at how it's used below (`if cctx.Args().Present()`). Let's use the hint again:

```suggestion
				if cctx.NArg() > 1 {
					return IncorrectNumArgsWithHint(cctx, "expected 1 at most")
				}
```